src/main/resources/reference.conf

```diff
@@ -16,6 +16,10 @@ gpalloc {
   #https://cloud.google.com/service-management/quotas
   opsThrottle = 15
   opsThrottlePerDuration = 10 seconds
+
+  # GCP quota for service accounts is 100. We delete any projects that have more than 90 pets to avoid this quota
+  # https://cloud.google.com/iam/quotas#quotas
+  maxPets = 90
```

Comment:
just out of curiosity, how many pets does a test run create?

Reply:
Good question. I'm not entirely sure how many pets a given test run creates, but we are banking on there not being any tests that create more than 10 pets for us to not hit quota issues. That is a pretty safe assumption based on all the tests I'm aware of, as a test would have to involve >10 users for it to make >10 pets. The one potential outlier that I could think of was that Leo reuses the same gpalloc project for a bunch of its integration tests because it reuses a single cluster within that project. With the same project being reused without getting released, it would be possible for Leo to create too many pets. Fortunately, since Leo creates a cluster per user per project, they are only using a single test user in the reused gpalloc project and only creating a single pet for that user so we're okay there too.